uportal-war/src/main/java/org/apereo/portal/layout/dlm/DistributedLayoutManager.java

```diff
@@ -452,9 +453,80 @@ public IUserLayoutNodeDescription addNode(
 
             return node;
         }
+
+        if (LOG.isTraceEnabled()) {
```

Comment:
- Suggest `DEBUG` level here
  - Do you have the bandwidth to convert this file to slf4j, since you're in here making changes?